_posts/2016-09-10-stripe-dress.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: The psychology of perception tests an age-old fashion foible
+author: neuroamanda
+image:
+  feature: 2016-09-10-stripe-dress.jpg
+  credit: Gareth Williams
+  caption: Horizontal-striped dress
+---
+
+_Horizontal stripes on clothing are in fact slimming_
+
+[The dress](https://en.wikipedia.org/wiki/The_dress) sparked the hottest fashion debate of the 21st century, but another mystery has occupied scientists and fashionistas alike since the mid-19th century: do horizontal stripes make you look fat?
+The answer may seem obvious, but experimental results and a well-documented illusion contradict most people's intuition that horizontal stripes make the wearer look wider.
```

Comment:
Changed most to many.
I use "popular belief" the very next line so I don't want to repeat myself.
The opener of the paper even says "It is commonly believed that horizontal stripes on clothing make them look fatter." And the link provided in para 2 says "Traditionally, it's been thought that vertical stripes make you look taller and thinner."